### PR TITLE
General options for alerts in the admin config

### PR DESCRIPTION
**Pull request by dkondyrev**

### CODE REVIEW

File: tests/fixtures/renderer_test_user_rules/test_rule_1.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - email:
+      email: test@gmail.com
+      email_config_name: mail_2
+  - email:
+      email: test2@gmail.com
+      email_config_name: default
+  - slack:
+      slack_username_override: test_user
+      slack_channel_override: test_channel
+      slack_config_name: slack_1
```
Comment (rvadim) [inline tests/fixtures/renderer_test_user_rules/test_rule_1.yaml]: Let's rename `<notifier>_config_name` to `<notifier>_id`

File: tests/fixtures/renderer_test_user_rules/test_rule_1.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      email: example@gmail.com
+      email_config_name: mail_1
+  - email:
+      email: test@gmail.com
+      email_config_name: mail_2
+  - email:
+      email: test2@gmail.com
+      email_config_name: default
+  - slack:
+      slack_username_override: test_user
```
Comment (rvadim) [inline tests/fixtures/renderer_test_user_rules/test_rule_1.yaml:16]: Why override? Let's use just `username` and `channel`
Also, can I use only `channel` or only `username`?
This is user interface it should be simpler

**Review by rvadim — CHANGES_REQUESTED** [review]
Good, but some questions about user interface.

File: tests/fixtures/renderer_test_user_rules/test_rule_1.yaml (hunk 1 above)
Comment (dkondyrev) [reply tests/fixtures/renderer_test_user_rules/test_rule_1.yaml]: Changed

File: tests/fixtures/renderer_test_user_rules/test_rule_1.yaml (hunk 2 above)
Comment (dkondyrev) [reply tests/fixtures/renderer_test_user_rules/test_rule_1.yaml:16]: I used standard ElastAlert option names. Should I add custom options and implement the generation of standard ElastAlert options?

**Review by Kuyantsev-VP — DISMISSED** [review]
LGTM

File: tests/fixtures/renderer_test_user_rules/test_rule_1.yaml (hunk 2 above)
Comment (rvadim) [reply tests/fixtures/renderer_test_user_rules/test_rule_1.yaml:16]: Let's add options mapping from user interface to rules file

Comment (Kuyantsev-VP) [reply tests/fixtures/renderer_test_user_rules/test_rule_1.yaml:16]: You mean adding appropriate options to jinja template to render them in EA rule files?

**Review by rvadim — DISMISSED** [review]
LGTM

**Review by rvadim — APPROVED** [review]

**Comment by Kuyantsev-VP** [comment]
LGTM

**Review by Kuyantsev-VP — APPROVED** [review]

**Pull request merged**